bool load
uint32[4] top_screen_x
uint32[4] top_screen_y
uint32[4] bottom_screen_x
uint32[4] bottom_screen_y
---
uint32[4] top_screen_x
uint32[4] top_screen_y
uint32[4] bottom_screen_x
uint32[4] bottom_screen_y